Source/WebKit/WebKitSwift/TextExtraction/WKTextExtractionItem.swift

```diff
@@ -22,75 +22,81 @@
 
-    public init(with rectInWebView: CGRect, children: [WKTextExtractionItem]) {
+@_objcImplementation extension WKTextExtractionItem {
+    let rectInWebView: CGRect
+    let children: [WKTextExtractionItem]
+
+    @objc
+    fileprivate init(with rectInWebView: CGRect, children: [WKTextExtractionItem]) {
         self.rectInWebView = rectInWebView
         self.children = children
     }
 }
 
-@available(iOS 17.0, macOS 12.0, *)
-@objc public enum WKTextExtractionContainer: Int {
-    case root
-    case viewportConstrained
-    case list
-    case listItem
-    case blockQuote
-    case article
-    case section
-    case nav
-    case button
-}
-
-@available(iOS 17.0, macOS 12.0, *)
-@objc(WKTextExtractionContainerItem) public class WKTextExtractionContainerItem: WKTextExtractionItem {
-    @objc public let container: WKTextExtractionContainer
+@_objcImplementation extension WKTextExtractionContainerItem {
+    let container: WKTextExtractionContainer
 
-    @objc public init(container: WKTextExtractionContainer, rectInWebView: CGRect, children: [WKTextExtractionItem]) {
+    init(container: WKTextExtractionContainer, rectInWebView: CGRect, children: [WKTextExtractionItem]) {
         self.container = container
         super.init(with: rectInWebView, children: children)
     }
 }
 
-@available(iOS 17.0, macOS 12.0, *)
-@objc(WKTextExtractionEditable) public class WKTextExtractionEditable: NSObject {
-    @objc public let label: String
-    @objc public let placeholder: String
-    @objc public let isSecure: Bool
-    @objc public let isFocused: Bool
+@_objcImplementation extension WKTextExtractionEditable {
+    let label: String
+    let placeholder: String
+
+    // Properties with a customized getter are incorrectly mapped when using ObjCImplementation.
+    @nonobjc private let backingIsSecure: Bool
+    @objc(secure)
+    var isSecure: Bool {
+        @objc(isSecure)
+        get { backingIsSecure }
+    }
 
-    @objc public init(label: String, placeholder: String, isSecure: Bool, isFocused: Bool) {
+    // Properties with a customized getter are incorrectly mapped when using ObjCImplementation.
+    @nonobjc private let backingIsFocused: Bool
+    @objc(focused)
+    var isFocused: Bool {
+        @objc(isFocused)
+        get { backingIsFocused }
+    }
+
+    init(label: String, placeholder: String, isSecure: Bool, isFocused: Bool) {
         self.label = label
         self.placeholder = placeholder
-        self.isSecure = isSecure
-        self.isFocused = isFocused
+        self.backingIsSecure = isSecure
+        self.backingIsFocused = isFocused
     }
 }
 
-@available(iOS 17.0, macOS 12.0, *)
-@objc(WKTextExtractionLink) public class WKTextExtractionLink: NSObject {
-    @objc public let url: NSURL
-    @objc public let range: NSRange
+@_objcImplementation extension WKTextExtractionLink {
+    // Used to workaround the fact that `@objc @implementation` does not support stored properties whose size can change
```

Comment:
I think this comment needs to be updated, since we're not actually using `@objc @implementation` yet?